app/src/main/AndroidManifest.xml

```diff
@@ -38,7 +38,7 @@
 		android:roundIcon="@mipmap/ic_launcher_round"
 		android:supportsRtl="true"
 		android:theme="@style/AppTheme.Light"
-		tools:ignore="AllowBackup,GoogleAppIndexingWarning">
+		tools:ignore="AllowBackup,GoogleAppIndexingWarning,UnusedAttribute">
```

Comment:
Which attribute is unused?

Reply:
`android:networkSecurityConfig` for older Android versions, but I guess that it is required for newer ones so we can't remove it here.

Reply:
strange, but if it is needed for newer versions, lint shouldn't complain about it!?

Reply:
Another option for lint to ignore this is adding `tools:targetApi="n"`. Apparently this really only affects lint. Do you want me to change it accordingly?

Reply:
Changed to `tools:targetApi="n"`.